app/pages/lab/subject-set.cjsx

```diff
@@ -17,9 +17,6 @@ VALID_SUBJECT_EXTENSIONS = ['.jpg', '.jpeg', '.png', '.gif', '.svg']
 INVALID_FILENAME_CHARS = ['/', '\\', ':', ',']
 MAX_FILE_SIZE = 600 * 1024
 
-announceSetChange = ->
-  apiClient.type('subject_sets').emit 'add-or-remove'
```

Comment:
git blame points to this commit for that line in classify.cjsx, if that's any help? https://github.com/zooniverse/Panoptes-Front-End/commit/00e607ae6ae60431861b3267d98be8a391825999